OpenIPSL/Electrical/Machines/PSSE/BaseClasses/baseMotor.mo

```diff
@@ -0,0 +1,129 @@
+within OpenIPSL.Electrical.Machines.PSSE.BaseClasses;
+partial model baseMotor "Base model for the PSSE three-phase induction motor models"
+
+  parameter OpenIPSL.Types.ApparentPower M_b = 15e6 "Machine base power"
+                                                                        annotation (Dialog(group="Power flow data"));
+  extends OpenIPSL.Electrical.Essentials.pfComponent(
+    final enabledisplayPF=false,
+    final enablefn=false,
+    final enableV_b=false,
+    final enableangle_0=false,
+    final enablev_0=false,
+    final enableQ_0=false,
+    final enableP_0=false,
+    final enableS_b=true);
+
+    import Modelica.Constants.pi;
```

Comment:
Please use `C.pi` instead which is available via the global import. So remove this import.